BeeSwift/ConfigureNotificationsViewController.swift

```diff
@@ -121,7 +121,7 @@ extension ConfigureNotificationsViewController : UITableViewDataSource, UITableV
     }
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueReusableCell(withIdentifier: self.cellReuseIdentifier) as! SettingsTableViewCell!
+        let cell = tableView.dequeueReusableCell(withIdentifier: self.cellReuseIdentifier) as! SettingsTableViewCell?
```

Comment:
The combination of as**!** and SettingsTableViewCell**?** seems to be contradictory. 

On the one hand, sure maybe we can assume these do not break. On the other, how about

```

guard
  let cell = tableView.dequeueReusableCell(withIdentifier: self.cellReuseIdentifier) as? SettingsTableViewCell
else
  { return UITableViewCell() }

```